oonib/txextra.py

```diff
@@ -0,0 +1,81 @@
+from twisted.web.http_headers import Headers
```

Comment:
tx is a convention used to indicate third-party "Twisted" extensions, for example `txtorcon`, `txsocksx`. I think the origin is "Twisted eXtension".